File: EasyReportDemo/Models/Customer.cs
using System.ComponentModel.DataAnnotations;
using Korzh.EasyQuery;

namespace EasyReportDemo.Models
{

    [DisplayColumn("Name")]
    [EqEntity(DisplayName = "Client")]
    public class Customer {
        public string Id { get; set; }

        [Display(Name = "Company Name")]
        public string CompanyName { get; set; }
        
        public string Street { get; set; }

        public string City { get; set; }

        public string Region { get; set; }

        public string PostalCode { get; set; }

        [EqListValueEditor]
        public string Country { get; set; }

        public string ContactName { get; set; }

        public string ContactTitle { get; set; }

        public string Phone { get; set; }

        public string Fax { get; set; }

    }



}

File: AspNet4/WebForms/VB.AdvancedSearch/Models/NWind/Customer.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports Korzh.EasyQuery

<DisplayColumn("Name")>
<EqEntity(DisplayName:="Client")>
Public Class Customer
    <Column("CustomerID")>
    Public Property Id As String
    <Display(Name:="Company Name")>
    Public Property CompanyName As String
    Public Property Address As String
    Public Property City As String
    Public Property Region As String
    Public Property PostalCode As String
    <EqListValueEditor>
    Public Property Country As String
    Public Property ContactName As String
    Public Property ContactTitle As String
    Public Property Phone As String
    Public Property Fax As String
End Class